setup.py

```diff
@@ -74,8 +74,7 @@
 
     # See https://pypi.python.org/pypi?%3Aaction=list_classifiers
     classifiers=[
-        'Development Status :: 5.3.1 - Beta',
-
+        'Development Status :: 4 - Beta',
```

Comment:
Should this also be 5.5.0?

Reply:
That is the list of classifier for pypi https://pypi.python.org/pypi?%3Aaction=list_classifiers. That just tells it its in beta. If we are out of Beta we should change it to `Development Status :: 5 - Production/Stable`

Reply:
Got you, let's keep it as beta for now. We will likely hit production level with JWT is implemented, that will be a community decision though.